Client/game_sa/CGameSA.cpp

```diff
@@ -265,6 +267,7 @@ CGameSA::~CGameSA()
     delete reinterpret_cast<CAEAudioHardwareSA*>(m_pAEAudioHardware);
     delete reinterpret_cast<CAudioContainerSA*>(m_pAudioContainer);
     delete reinterpret_cast<CPointLightsSA*>(m_pPointLights);
+    delete static_cast<CColStoreSA*>(m_collisionStore);
```

Comment:
Why even is this cast needed?

Reply:
CColStore class is not using a virtual destructor currently and the member variable is stored as `CColStore*` to keep the code a little bit simpler. But I am going to make the destructor virtual.